public/components/query_compare/create_index.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiPageBody, EuiEmptyPrompt, EuiLink } from '@elastic/eui';
+
+import { Header } from '../common/header/header';
+
+export const CreateIndex = () => {
+  return (
+    <>
+      <Header />
+      <EuiPageBody>
+        <EuiEmptyPrompt
+          title={<h2>Create an index to start comparing search results. </h2>}
+          body={
+            <p>
+              Before you can query data, you have to index it.{' '}
+              <EuiLink>Learn how to index your data.</EuiLink>
+            </p>
+          }
+        />
+      </EuiPageBody>
+    </>
+  );
```

Comment:
Is this going to be part of the final plugin? Or was this copied over from somewhere? 

In practice, I don't think anyone would want to use the query comparison plugin to create a new index.

Reply:
This page is from the mock. User will get into this page if there is no index exists.

Reply:
> Can we resolve this as is?

The only change I might make would be renaming the file to something like `no_available_indexes.tsx`. The purpose of the page isn't to create an index, but rather to serve a useful error page if there are no indexes.

Reply:
> The only change I might make would be renaming the file to something like `no_available_indexes.tsx`. 

I agree with the new file name. 

Speaking with available indexes, should we let users select all indexes returned from [this call](https://github.com/mingkunm/dashboards-search-relevance/blob/48ddab67aad0c4085a2aac97074f50f72d81ffd1/server/routes/dsl_route.ts#L43)? I saw individual index has `health` and `status` attributes. Do we want to display all of them in Select Index Dropdown?

![image](https://user-images.githubusercontent.com/55871308/198400003-62ecd0cc-4b78-48ec-9e34-4f59928fc64d.png)

Reply:
>  I saw individual index has health and status attributes. Do we want to display all of them in Select Index Dropdown?

I think for now, we can keep it simple and show all indices returned from `/_cat/indices`. If an index is unable to handle a search request, we should surface an error. An index is "red" state is not usual, so IMO we don't need to worry about it too much for now.

In the long run, maybe we'll hide the indexes that start with `.` by default? Again, not something I would worry about for now.